clp_ffi_py/ir/readers.py

```diff
@@ -21,17 +22,30 @@ class ClpIrStreamReader(Iterator[LogEvent]):
     :param allow_incomplete_stream: If set to `True`, an incomplete CLP IR stream is not treated as
         an error. Instead, encountering such a stream is seen as reaching its end without raising
         any exceptions.
+    :param decoder_buffer_size: Deprecated in 0.0.13. The purpose of this argument is to provide
+        backward compatibility. It overwrites `deserializer_buffer_size`'s value if not None.
```

Comment:
```suggestion
    :param decoder_buffer_size: Deprecated since 0.0.13. Use `deserializer_buffer_size` instead.
        This argument is provided for backward compatibility and if set, will overwrite
        `deserializer_buffer_size`'s value.
```